AirForSteam.json

```diff
@@ -2,34 +2,39 @@
     "version": "2018-0406",
     "description": "An elegant and customizable skin for Steam.",
     "homepage": "http://airforsteam.com/",
-    "license": " CC-BY-NC-4.0",
+    "license": {
+        "identifier": "CC-BY-NC-4.0",
+        "url": "https://github.com/Outsetini/Air-for-Steam/blob/master/LICENSE.txt"
```

Comment:
If it’s an [SPDX](https://spdx.org/licenses/) license, I’m not including a URL, as it’s a standard license.

Reply:
I am including URL whener it's possible and when licence file exists. I prefer to cite and document everything.

Reply:
I would prefer that, as well, but don't have the time to document the 800+ apps under scoop/extras/versions!

Reply:
Feel your pain :/